exercise/0-IH-exercise/main.go

```diff
@@ -1,9 +1,54 @@
 package main
 
+import (
+	"fmt"
+	"runtime"
+	"time"
+)
+
+
 func main() {
-	// 3 task concurrent
-	// pertask beda2 sleepnya
-	// gunakan wait group dan channel
-	// waitnya tidak menunggu dari 5 detik
-	// cancelation bisa menggunakn time, context, channel
+	// use channel
+
+	runtime.GOMAXPROCS(2)
+
+	timeStart := time.Now()
+
+	var messages = make(chan string)
```

Comment:
if we can predict how many data will get into the channel, better to use buffered channel.